test/remoteIncludes.spec.js

```diff
@@ -11,10 +11,10 @@ let harvester = require('../lib/harvester');
 
 let harvesterOptions = {
   adapter: 'mongodb',
-  connectionString: 'mongodb://127.0.0.1:27017/testDB',
+  connectionString: process.env.MONGODB_URL || 'mongodb://db:27017/testDB',
```

Comment:
db is an alias for the container ip in which mongodb is running. It should play well with non-dockeried users as long as they pass their mongodb connection string to the MONGODB_URL env var.